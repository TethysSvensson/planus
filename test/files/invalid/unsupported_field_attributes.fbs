struct Unsupported {
  v0: uint8 (force_align: 16);
  v1: uint8 (key);
}
