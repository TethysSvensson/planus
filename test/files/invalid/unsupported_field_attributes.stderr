error: Metadata attribute does not make sense on struct fields
  ┌─ test/files/invalid/unsupported_field_attributes.fbs:2:14
  │
2 │   v0: uint8 (force_align: 16);
  │              ^^^^^^^^^^^^^^^

error: Metadata attribute is not currently supported
  ┌─ test/files/invalid/unsupported_field_attributes.fbs:3:14
  │
3 │   v1: uint8 (key);
  │              ^^^

Error: Bailing because of errors
